pages/blog/conference-2023.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "AsyncAPI Conference on Tour 2023 Report"
+date: 2024-01-25T06:00:00+01:00
+type: Communication
+tags:
+  - Conference
+cover: /img/posts/2023-summary/background.webp
+authors:
+  - name: Thulisile Sibanda
+    photo: /img/avatars/thulieblack.webp
+    link: https://www.linkedin.com/in/v-thulisile-sibanda/
+    byline: AsyncAPI Community Manager
+excerpt: In 2023, we hosted our very first in-person conference edition. Find out how it all went down!
+---
```

Comment:
make it featured please

probably some other arts should no longer be featured